### ISSUE TITLE
Linux Mint bug

### ISSUE THREAD

**Issue by metarhyme** [adobe/brackets#12936]
O.S. Linux Mint 18 'Sarah' "Xfce 64-bit" - installation problem

http://brackets.io correctly selects Brackets.Release.1.8.64-bit.deb which downloads, however a libgcrypt11 unsatisfied dependency prevented installation by software manager.  By installing Transitional libgcrypt11-dev and then using the via PPA for Ubuntu instruction at
http://www.webupd8.org/2013/11/install-brackets-in-ubuntu-via-ppa-open.html Brackets did install.

I suppose it's possible that brackets really ought to be dependent on libgcrypt20, which is current, rather than libgcrypt11, which seems to be deprecated. 
![brackets](https://cloud.githubusercontent.com/assets/23566296/20544034/c4e00d78-b0cd-11e6-8fb0-ac3b173c0920.png)

**Comment by zaggino** [comment]
@NathanJPlummer wrote a guide for troubleshooting Linux which is linked in README - https://nathanjplummer.github.io/Brackets/

This is a known issue

**Comment by metarhyme** [comment]
Thanks for the link to doing what I did, almost - I hope it helps others.  Attempting to install 1.8.64-bit.deb after Transitional libgcrypt11-dev obtained and installed by software manager also didn't work - I had to go the PPA command line route.  After Brackets installed I tried to uninstall the dummy Transitional libgcrypt11-dev obtained by software manager from the Mint repository but that hung at 75%, so I told software manager to install it, which it did.
